pyboy/openai_gym.py

```diff
@@ -23,43 +23,27 @@ class Env:
 class PyBoyGymEnv(Env):
     """ A gym environement built from a `pyboy.PyBoy`
 
-    Arguments
-    ---------
+    This function requires PyBoy to implement a Game Wrapper for the loaded ROM. You can find the supported games in pyboy.plugins.
 
-        pyboy: `pyboy.PyBoy`
-            A PyBoy game instance
+    Args:
+        observation_type (str): Define what the agent will be able to see:
+        * `"raw"`: Gives the raw pixels color
+        * `"tiles"`:  Gives the id of the sprites in 8x8 pixel zones of the game_area defined by the game_wrapper.
+        * `"compressed"`: Gives only the id of used grouped sprites defined by the game_wrapper.
+        * `"minimal"`: Gives a minimal representation defined by the game_wrapper (recommended).
```

Comment:
I think these explanations could be improved. I don't exactly get the difference between the two.

Reply:
Well it's a level of detail, compressed is less detailled than tiles but still there is a lot of informations (Unique enemies, objects and so on) and minimal is the strict minimum of information to have an algorithm perform anything.

I'm not sure how to describe that properly but Mario wrapper is a good example

Reply:
> I'm not sure how to describe that properly but Mario wrapper is a good example

In Mario minimal has a size of 5 ('Empty', 'Mario', 'Bonuses', 'Blocks', 'Enemies') and compressed has a size of 28.

Reply:
I tired * `"compressed"`: Gives a more detailled but heavier representation than `"minimal"`. is that better ?